app/models/form/question.rb

```diff
@@ -73,19 +75,45 @@ def has_inferred_check_answers_value?(case_log)
 
   def update_answer_link_name(case_log)
     link_type = if type == "checkbox"
-                  answer_options.keys.any? { |key| case_log[key] == "Yes" } ? "Change" : "Answer"
+                  answer_options.keys.any? { |key| case_log[key] == 1 } ? "Change" : "Answer"
```

Comment:
Is this always true? I thought some answers could be Yes with a different value than 1.

Reply:
Good point